es-ds-docs/pages/molecules/progress-circle.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <div>
+        <h1>
+            Progress circle
+        </h1>
+        <p class="pb-200">
+            Extended from <b-link
+                href="https://github.com/EnergySage/es-cdgm/blob/main/frontend/components/CircularProgress.vue"
+                target="_blank">
+                <code>es-cdgm</code> codebase
+            </b-link>
+        </p>
+        <es-progress-circle
+            class="mb-100"
+            :value="value"
+            height="100px" />
+        <div class="d-flex flex-nowrap">
+            <es-button
+                size="sm"
+                class="mr-50"
+                @click="value = value - 10">
+                -10%
+            </es-button>
+            <es-button
+                size="sm"
+                @click="value = value + 10">
+                +10%
+            </es-button>
+        </div>
```

Comment:
Could we add a props table here as well? This would be an improvement over the old one!